uarch/uarch-machine-state-access.h

```diff
@@ -522,8 +522,8 @@ class uarch_machine_state_access : public i_state_access<uarch_machine_state_acc
         return raw_read_memory<uint64_t>(shadow_state_get_csr_abs_addr(shadow_state_csr::htif_iyield));
     }
 
-    uint64_t do_poll_console(uint64_t mcycle) {
-        return mcycle;
+    std::pair<uint64_t, bool> do_poll_external_interrupts(uint64_t mcycle, uint64_t mcycle_max) {
```

Comment:
So the idea is really for uarch to operate as though virtio didn't exist?

Reply:
Correct, the idea is to not ever support any VirtIO device in uarch.

Reply:
We should probably check for the nrep and bail out from uarch with an error? Just to be safe?